portality/templates/doaj/article.html

```diff
@@ -25,6 +25,7 @@
 
 {% if jtitle %}<meta name="citation_journal_title" content="{{ jtitle }}">{% endif %}
 {% if jbib.publisher %}<meta name="citation_publisher" content="{{ jbib.publisher }}">{% endif %}
+{% if doi %}<meta name="citation_doi" content="{{ doi }}">{% endif %}
```

Comment:
Just a thought, we should consider normalising this DOI like we have done in the other PR https://github.com/DOAJ/doaj/pull/1650

We could use the `doi_url` filter here once that PR has been merged, which will give us a normalised DOI url.  The alternative is to use `portality.lib.normalise.normalise_doi` to generate a canonical form of it (which won't be a URL but will be an industry standard DOI).

If we take the latter option note that `normalise_doi` can throw an exception, and jinja2 has no exception framework, so we might need to add a template filter wrapper to it like `doi_url` to catch and manage that exception.